docs/pages/build/building-from-github.mdx

```diff
@@ -109,3 +116,101 @@ of the build in the PR's checks. A link to the build will be available in the ch
+and number of characters inside the pattern. For example, `releases/*` can match `releases/`,
+`release/1234`, `release/genesis`, and so on. If you specify the pattern as a sole asterisk (`*`),
+all branches/tags will be matched.
+
+<ImageSpotlight
+  alt="The default state of the new build trigger form"
+  src="/static/images/eas-build/build-from-github/empty-build-trigger-form.png"
+/>
+
+You can also configure triggers for specific platforms and build profiles. If you select multiple
+platforms, a separate trigger will be made for each.
+
+<ImageSpotlight
+  alt="A filled-out version of the new build trigger form"
+  src="/static/images/eas-build/build-from-github/filled-build-trigger-form.png"
+/>
+
+<ImageSpotlight
+  alt="The build triggers section on the Expo project GitHub settings page filled with build triggers"
+  src="/static/images/eas-build/build-from-github/filled-build-triggers-table.png"
+/>
+
+When you push to a branch or tag, you can find the builds by looking at the “Checks” section on a
+commit.
+
+<ImageSpotlight
+  alt="The GitHub checks section on a branch commit"
+  src="/static/images/eas-build/build-from-github/builds-executed-automatically-on-branch.png"
+/>
+
+<ImageSpotlight
+  alt="The GitHub checks section on a tag commit"
+  src="/static/images/eas-build/build-from-github/tag-triggered-build.png"
+/>
+
+For pull requests, you can configure a “target branch pattern”. This is the destination branch of
+the pull request you want to build. The same rules apply for wildcards here as well.
+
+<ImageSpotlight
+  alt="The build trigger form for pull request"
+  src="/static/images/eas-build/build-from-github/pull-request-trigger-form.png"
+/>
+
+When you push to a pull request with a source and target branch matching this trigger, you'll find
+these builds in the checks section of the pull request:
+
+<ImageSpotlight
+  alt="The GitHub checks section on a pull request"
+  src="/static/images/eas-build/build-from-github/pull-request-triggered-build.png"
+/>
+
+> **Note:** To trigger builds from a pull request, the pull request's author must be a collaborator
+> on the GitHub repository. If you want to build pull requests from external contributors, [apply a PR Label](#build-using-github-pr-labels).
+
+#### Managing build triggers
+
+On your project's GitHub settings page on expo.dev, you can click the options button to the right of
+a build trigger row to manage that trigger. You can disable, edit, or delete the trigger from here.
+
+<ImageSpotlight
+  alt="Manage GitHub build trigger with the options button"
+  src="/static/images/eas-build/build-from-github/managing-build-triggers.png"
+/>
+
+### Troubleshooting
+
+- When things go wrong, we'll comment on the commit we attempted to build with some error information. We also show the latest result in the build triggers UI, which includes error information when you hover the “Error” tag.
+
+<ImageSpotlight
```

Comment:
This image is too big. I'd also suggest taking it at 2x size.

![CleanShot 2023-11-22 at 22 10 00@2x](https://github.com/expo/expo/assets/10234615/3f5022e5-e084-4ff4-b995-f85c9ae6559b)

Reply:
Or we can use `maxWidth` style, like we do for few smaller images:
https://github.com/expo/expo/blob/2c346e0f00eee1ca8c7fa5df8287f0a796fb1dba/docs/pages/tutorial/build-a-screen.mdx#L36-L36